minimal/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "react-native-fullstack-boilerplate",
+  "version": "0.1.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "private": true,
+  "scripts": {
+    "start": "react-native-scripts start",
+    "exp:start": "exp start --lan",
```

Comment:
What does `--lan` afford users versus `react-native-scripts start`?

Reply:
Since we are depending on local ip rather than domain url, it's necessary to use --lan to make that app has access to prisma server